assessment.md

```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
+  * Excellent = Properly sets up all appropriate security features on project deliverables in way that is best-suited for given project deliverables, and actively monitors project for potential security threats or improvements.
 
 * Creating third-party integrations with GitHub Actions CI (e.g., codecov.io)
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not create third-party integrations with GitHub Actions CI.  
+  * Inadequate = Is able to occasionally integrate GitHub with a third party. Still lacks fundamental knowledge of how to do so consistently
+  * Adequate = Contributes third party integrations on a fairly consistent basis. Could use more knowledge on the subject, however.
+  * Good = Often times integrates third party software with Github. Very knowledgeable on how to do so.
+  * Excellent = Leads the way for the team when integrating Github with a third party. Can easily explain to the team how they did so.
 
 * Adding README badges that show status of GitHub Actions CI builds
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not add README badges.
+  * Inadequate = Rarely adds README badges that show status of builds, never addressing any errors made.
+  * Adequate = Occasionally adds README badges that show status of builds, rarely addressing errors any errors made.
+  * Good = Frequently adds README badges that show status of builds, often addressing any errors made.
+  * Excellent = Adds README badges that show status of builds, always addressing any errors made.
 
 * Adding README badges that show project characteristics (e.g., coverage and
   language)
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not add README badges.
+  * Inadequate = Rarely adds README badges that show project characteristics, never addressing any errors made.
+  * Adequate = Occasionally adds README badges that show project characteristics, rarely addressing errors any errors made.
+  * Good = Frequently adds README badges that show that show project characteristics, often addressing any errors made.
+  * Excellent = Adds README badges that show project characteristics, always addressing any errors made.
 
 * Identifying and resolving problems with failing builds in GitHub Actions CI
-
-TODO: add description for each assessment level for this skill.
+  * None = Ignores all problems in GitHub Actions CI, routinely makes and does not resolve basic coding errors.
+  * Inadequate = Occasionally identifies problems with builds, does not solve problems.
+  * Adequate = Frequently identifies problems, occasionally solves them partially.
+  * Good = Frequently identifies problems and makes strides to solve them at least partially. Occasionally provides documentation.
+  * Excellent = Routinely identifies and solves the problems behind failing builds and documents any fixes for future use.
```

Comment:
Instead of assessing the frequency of "identifying and resolving problems", I would advise the team to maybe address the percentage of problems that an individual is capable of identifying and resolving, i.e. identify some problems, identify most problems. Also, I think it would be great if you could be consistent with the use of "resolve" and "solve".